game/graphics/opengl_renderer/OpenGLRenderer.cpp

```diff
@@ -414,6 +414,9 @@ void OpenGLRenderer::setup_frame(int window_width_px,
   glDepthMask(GL_TRUE);
   glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT | GL_STENCIL_BUFFER_BIT);
   glDisable(GL_BLEND);
+
+  m_render_state.window_width_px = window_width_px;
+  m_render_state.window_height_px = window_height_px;
```

Comment:
I think I'm running into issues related to this, when the game doesn't take up the full window:
![image](https://user-images.githubusercontent.com/48171810/178080243-79c78ed4-64dc-49b3-95c2-9d3c5bedcf61.png)

I'm not actually sure this is something that's supposed to be possible or not, but the resolution/aspect ratio settings don't really work for me in the first place.  This is probably a separate issue and shouldn't block this PR.

Reply:
Turned out to be a simple issue with the framebuffer blit. The source dimensions needed to respect the XY offset of the renderer.